src/conf_mode/interfaces-nat64.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2019-2021 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import shutil
+from sys import exit
+
+from vyos.config import Config
+from vyos.configdict import get_interface_dict
+from vyos.configverify import verify_vrf
+from vyos.configverify import verify_address
+from vyos.ifconfig import NAT64If
+from vyos import ConfigError
+from vyos import airbag
+from vyos.template import render
+from vyos.util import call
+airbag.enable()
+
+
+def get_config(config=None):
+    """
+    Retrive CLI config as dictionary. Dictionary can never be empty, as at least the
+    interface name will be added or a deleted flag
+    """
+    if config:
+        conf = config
+    else:
+        conf = Config()
+    base = ['interfaces', 'nat64']
+    nat64 = get_interface_dict(conf, base)
+    return nat64
+
+def verify(nat64):
+    if 'deleted' in nat64:
+        # bail out early
+        return None
+
+    verify_vrf(nat64)
```

Comment:
I see no use of a VRF - I guess its a copy/paste issue ...

Reply:
Well we should start low and not asume VRF works out of the box. It needs to be tested and could be added later on. In general I like the idea of a per VRF nat64 gateway - but we need to check if the daemon supports it. Please drop:
- netns
- mirror
- redirect